cns/client_test.go

```diff
@@ -1232,8 +1235,8 @@ func TestClient(t *testing.T) {
 			Vm:            nodeVM.Reference(),
 			DiskMode:      "independent_persistent",
 			Sharing:       "sharingMultiWriter",
-			UnitNumber:    23,
-			ControllerKey: 1000,
+			UnitNumber:    &unitNumber,
+			ControllerKey: &controllerKey,
```

Comment:
I think you can do:
```
			UnitNumber:    vim25types.NewInt32(23),
			ControllerKey: vim25types.NewInt32(1002),
```
rather than declaring variables just to get a pointer out of it.